src/reducers/registration.test.js

```diff
@@ -0,0 +1,20 @@
+import { userConstants } from '../constants';
+import { registration } from './registration.reducer';
+
+describe('Registration Reducer', () => {
+  test('check initiation of state', () => {
+    expect(registration(undefined, {})).toEqual({});
+  });
+  test('check register_request', () => {
+    expect(registration({}, { type: userConstants.REGISTER_REQUEST }))
```

Comment:
use variables and spell things out to make your code more readable.

a typical format for unit testing is:
```
const expected = "whatever result you expect here"
const actual = theFunctionCallsReturnValueYouAreTestingHere(testParamsHere);
```

see an example from my code that I wrote yesterday
https://www.dropbox.com/s/h3r8rg6c1png0al/Screenshot%202018-07-09%2008.41.56.png?dl=0